Client/Frontend/Home/RemoteTabsPanel.swift

```diff
@@ -63,6 +64,15 @@ class RemoteTabsPanel: UITableViewController, HomePanel {
         self.SELrefresh()
     }
 
+    override func viewDidAppear(animated: Bool) {
```

Comment:
Can we split this into a separate bug? It has some potential side-effects, and I'd like to make sure we can land just the stringy parts.